kanimal/Writer/ScmlWriter.cs

```diff
@@ -52,7 +52,12 @@ public override OutputFiles Save()
             {
                 var stream = new MemoryStream();
                 sprite.Bitmap.Save(stream, ImageFormat.Png);
-                files[sprite.Name + ".png"] = stream;
+                var outputName = sprite.Name;
```

Comment:
this bit of code is a workaround for the `.png.png` file ending when converting from scml to scml that I mentioned earlier. It may not really be supposed to be in this pull request but I felt it was necessary for testing so that users of this pull request could check the interpolated scml.